### PR TITLE
build: update to latest jasmine and resolve errors

### PR DESCRIPTION
**Pull request by crisbeto**
Updates to the latest version of Jasmine and fixes some lint failures.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "@types/glob": "^7.1.3",
     "@types/gulp": "4.0.8",
     "@types/inquirer": "^7.3.1",
-    "@types/jasmine": "^3.5.4",
+    "@types/jasmine": "^3.6.3",
```
Comment (josephperrott) [inline package.json:103]: Can we have our `@types/jasmine` and  and `jasmine` versions match?

**Review by josephperrott — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (crisbeto) [reply package.json]: Done, although there's nothing actually keeping them in sync on npm.

Comment (josephperrott) [reply package.json]: Oh interesting, I was under the understanding they were  in lockstep.

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request merged**